scripts/map-randomizer.js

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
The script tag was wrapped with an IIFE to prevent the introduction of unwanted global function to the global namespace of JavaScript, that should probably be returned as you're currently introducing all defined functions as globals which can clash when interacting with the DOM.

Reply:
Cool, learning something new already!

So should I then move the `shuffleArray()` function into the IIFE?

Reply:
Yep. But I already see that you've done that!